packages/graphql/tests/integration/aggregations/where/edge/int.int.test.ts

```diff
@@ -751,4 +751,79 @@ describe("aggregations-where-edge-int", () => {
             }
         });
     });
+
+    describe("sum", () => {
+        test("should return posts where the sum of a edge like Int's is EQUAL to", async () => {
+            const session = driver.session();
+
+            const typeDefs = `
+                type User {
+                    testString: String!
+                }
+              
+                type Post {
+                  testString: String!
+                  likes: [User] @relationship(type: "LIKES", direction: IN, properties: "Likes")
+                }
+    
+                interface Likes {
+                    someInt: Int
+                }
+            `;
+
+            const testString = generate({
+                charset: "alphabetic",
+                readable: true,
+            });
+
+            const someInt1 = Math.floor(Math.random() * Math.random());
+            const someInt2 = Math.floor(Math.random() * Math.random());
+            const someInt3 = Math.floor(Math.random() * Math.random());
+
```

Comment:
I'm not sure what is the advantage of using random data here :confused:, I think random data adds unnecessary complexity to a test.

In fact, I think this setup is already problematic, as `Math.random()` gives a value between 0 and 0.99999..., multiplied with another `Math.random` we will always get a value between 0 and 0.999...,  using `.floor` means that `someInt1`, `someInt2`, `someInt3` are **always** 0

IMO, random data in tests make repeatability and readability harder (+ prone to bugs such as this one), without really giving any wider range of testing (with exceptions, of course), at best you get the same test as hard-coded data, at worst you get a test that randomly fails.

Reply:
Given Dan's absence this week, the fact that the tests pass and the fact that this is littered _everywhere_, let's leave this as it is for now and refactor all instances of this in a different branch? I've created a card in Trello for this. I also think we could be making better use of holistic test data generators such as `faker`, instead of using `generate` as we currently do.